packages/dx-react-scheduler/docs/guides/fundamentals.md

```diff
@@ -0,0 +1,44 @@
+# React Scheduler - Fundamentals
+
+The [DevExtreme React Scheduler](https://devexpress.github.io/devextreme-reactive/react/scheduler) allow you to display a structure of appointments and provide a functionality to manage data.
+
+## Overview
+
+This React Scheduler has built-in functionality for appointment layouting in many available time views. The scheduler has ability to control basic operations such as date navigation, view switching and data visualization features like [All-Day Panel](../reference/all-day-panel.md), [Appointment Tooltip](../reference/appointment-tooltip.md), [Appointment Form](../reference/appointment-form.md) and others. Also this widget provide functionality to [data editing](../reference/editing-state.md) such as creating, updating and deleting, and UI components with special input forms and editing abilities.
+
+The following image is an element map. It displays the basics concepts of the React Scheduler.
+
+<p align="center">
+  <img class="img-responsive" src="../../img/scheduler-elements.png">
+</p>
+
+In the image above demonstrates features that can be applied by adding special plugins.
+
+- [ViewSwitcher](../reference/view-switcher.md) - allow quick switching between views
+- [DateNavigator](../reference/date-navigator.md) - allow change current date and show visible time interval
+- [AllDayPanel](../reference/all-day-panel.md) - allow display longer then a day appointments in a specific place
+- [AppointmentTooltip](../reference/appointment-tooltip.md) - allow display short information about event in tooltip form also provide a some UI functionality to fast data editing.
+
```

Comment:
> allow quick switching
> allow change current

Why do you always use 'allow'? `ViewSwitcher` changes the current view. `DateNavigator` changes the current date, `AlDayPanel` displays all-day-appointments etc.

Reply:
> provide a some UI functionality

**a** or **some** ?))

Reply:
fast -> quick